.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Nimble Install
         shell: bash
         run: |
-          nimble -y install
+          nimble -y install -l
```

Comment:
we should be transitioning to `setup` here, as a longer-term plan since it does a better job of isolating the workspace

Reply:
How so? The only thing it does is to add the `config.nims` which is not needed by `nimble` anyways. No?